src/components/FeaturedArts.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import FeatureCard from './FeatureCard.jsx';
+import Arrow from './FeatureArrow.jsx';
+import {
+  nextSlide,
+  previousSlide,
+  setInitData,
+} from '../actions/featureActions';
+
+/**
+ * @param {class} component
+ *  @returns {JSX} jsx
+ */
+export class FeaturedArts extends Component {
+  /**
+   * @description handles state change
+     * @returns {state} current state
+     */
+  componentDidMount() {
+    this.props.setInitData();
+    setInterval(this.props.nextSlide, 5000);
+  }
+
+  /**
+   * @param {function} render
+   *  @returns {JSX} jsx
+   */
+  render() {
+    return (
+      <div>
+      <div id="feature_list">
+        <div id="featured_arts_header">
+          <h5>Featured Arts</h5>
+          <hr/>
+        </div>
+        {
+          this.props.data.activeList.map(
```

Comment:
To keep this clean, you can abstract this to a class method.

Reply:
Done